django/contrib/admin/options.py

```diff
@@ -1836,7 +1851,21 @@ def _changeform_view(self, request, object_id, form_url, extra_context):
         ModelForm = self.get_form(
             request, obj, change=not add, fields=flatten_fieldsets(fieldsets)
         )
+
+        actions = self.get_actions(request)
         if request.method == "POST":
+            selected_action = request.POST.get("action", "")
+            if actions and selected_action:
```

Comment:
I wonder if we could use walrus operator here?

Reply:
I think yes! As it is used just for the condition! Thank you!